v3/util/jenkins_update_config.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+JENKINS_DIRECTORY=$1
+
+USAGE_MESSAGE="Control Jenkins: Please provide the Jenkins home directory"
+
+if [[ ! $1 ]]; then
+  echo "$USAGE_MESSAGE"
+  exit 1;
+fi
+
+if [[ -n $CONTROL_JENKINS_OKTA_METADATA ]] ; then 
```

Comment:
This looks good, but pending our discussion for https://github.com/adobe-platform/infrastructure/pull/34, this might look like:
`etcd get /jenkins/oktametadata`
or similar.